tests/e2e-legacy/runTestForTags.mjs

```diff
@@ -0,0 +1,27 @@
+import { join, resolve } from "path";
+import { spawn } from "child_process";
+import { getDirName } from "./getDirName.mjs";
+
+const __dirname = getDirName();
+const ROOT = resolve(join(__dirname, "..", ".."));
+
+export const runTestForTags = (tagsToTest) => {
+  if (tagsToTest.length === 0) {
+    console.info("No clients with integration test cases have changed.");
+    return;
+  }
+
+  // Cucumber requires cwd to contain the test cases.
+  const command = `${join("node_modules", ".bin", "cucumber-js")}`;
+  const args = ["--fail-fast", "-t", `"${tagsToTest.join(" or ")}"`];
+  console.info(`Running cucumber test: \n${command} ${args.join(" ")}`);
+
+  const execOptions = { ...process, cwd: ROOT, encoding: "utf-8", shell: true };
+  const cucumber = spawn(command, args, execOptions);
+  cucumber.stdout.pipe(process.stdout);
+  cucumber.stderr.pipe(process.stderr);
+  cucumber.on("close", (code) => {
+    if (code === 0) process.exit();
+    else process.exit(code);
```

Comment:
since `process.exit(0)` is fine too, why not run `process.exit(code)` for all cases?

Reply:
From existing code: https://github.com/aws/aws-sdk-js-v3/blob/021f2181d3fa7d3a2760c97b66ab76f68c70e2f6/tests/e2e-legacy/index.js#L46-L49

Updated in [`cc02ac9` (#6538)](https://github.com/aws/aws-sdk-js-v3/pull/6538/commits/cc02ac9f842472d3631092275ac9b1e343f25c54)